eth_tester/backends/pyethereum/v16/main.py

```diff
@@ -48,6 +49,17 @@
 from eth_tester.backends.pyethereum.validation import (
     validate_transaction,
 )
+from eth_tester.utils.formatting import (
+    replace_exceptions,
+)
+
+
+if is_pyethereum16_available():
+    from ethereum.tester import (
+        TransactionFailed as Pyeth16TransactionFailed,
+    )
+else:
+    Pyeth16TransactionFailed = Exception
```

Comment:
Eh, I think this should be set to `None`, otherwise it will replace *all* exceptions with a bare `Exception` which I think will obfuscate errors we don't want to intercept.